### PR TITLE
Set balance to 0

### PR DESCRIPTION
**Pull request by minsulee2**

### CODE REVIEW

File: handlers/openai_ainize_handlers.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ class OpenaiAinizeHandler {
       }
 
       const ainizeResponse = await AinizeUtils.handleRequest({
-          req, amount: 0.1, ainizeStatus: AINIZE_STATUS.SUCCESS, responseData: response.data });
+          req, amount: 0, ainizeStatus: AINIZE_STATUS.SUCCESS, responseData: response.data });
```
Comment (platfowner) [inline handlers/openai_ainize_handlers.js:93]: Could you add a TODO for the price adjustment in the future?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: handlers/openai_ainize_handlers.js (hunk 1 above)
Comment (minsulee2) [reply handlers/openai_ainize_handlers.js:94]: Left the comment. Thanks!

**Comment by minsulee2** [comment]
Thanks! Merging now

**Pull request merged**